stargate/crates/graphql-parser/src/schema/format.rs

```diff
@@ -420,3 +423,23 @@ impl_display!(
     InputObjectTypeExtension,
     DirectiveDefinition,
 );
+
+#[cfg(test)]
+mod test {
+    use super::Document;
+    use crate::format::Style;
+    use crate::schema::grammar::parse_schema;
+
+    fn ast(s: &str) -> Document {
+        parse_schema(&s).unwrap()
+    }
+
+    #[test]
+    fn directive() {
+        assert_eq!(
+            ast("directive @key(fields: String!, graph: String!) repeatable on OBJECT")
+                .format(&Style::default()),
+            "directive @key(fields: String!, graph: String!) repeatable on OBJECT\n"
+        );
+    }
+}
```

Comment:
@queerviolet I didn't see a place where the formatter is tested. Feedback welcome on the approach here!

Reply:
We do round trip tests on the entire test corpus (ensuring the formatted output == the input file content), so this should already be tested through that. You could try breaking the formatter and ensuring the round trip test fails.